package.json

```diff
@@ -24,9 +24,9 @@
   },
   "dependencies": {
     "pg": "^6.1.3",
-    "sequelize": "^3.30.2"
+    "sequelize": "3.30.2 - 4.2"
```

Comment:
Should this be a dependency if it's also listed as a peer dependency? My understanding of peer dependencies is a little shaky but I was under the impression that having it as a peer dep was to explicitly avoid listing it as an direct dep? I think it probably won't make a difference because the specifications of the two dependencies are the same (and hence should get deduped by any modern-ish `npm`) but it seems odd to have both.